microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -31,6 +31,8 @@ When hosting the WebView2 control, your app has access to the following features
 
 <!-- maintenance notes: add table rows for any new h2 sections -->
 
+This page only lists Release APIs; it doesn't list Experimental APIs or Prerelease-stable APIs (that is, APIs which reached Stable status in a Prerelease SDK, but haven't yet been included in a Release SDK).  For Experimental APIs and Prerelease-stable APIs, see [Release Notes for the WebView2 SDK](../release-notes.md).
```

Comment:
@novac42 @peiche-jessica @nishitha-burman - pls review status names

Reply:
```suggestion
This page only lists APIs in Release SDKs; it doesn't list Experimental APIs or Stable APIs which are yet available in Prerelease SDKs.  For a comprehensive list of APIs including Experimental, see [Release Notes for the WebView2 SDK](../release-notes.md).
```
I'm more inclined to only describing APIs with "stable" and "experimental", and SDKs/packages with "release" and "prerelease".

Reply:
Using @peiche-jessica's wording, except have to change from:
Stable APIs which are yet available in Prerelease SDKs.
to either of:
Stable APIs which are not yet available in Release SDKs <--
Stable APIs which are still only available in Prerelease SDKs
so gives:
This page only lists APIs in Release SDKs; it doesn't list Experimental APIs, or Stable APIs which are not yet available in Release SDKs.  For a comprehensive list of APIs including Experimental APIs, see [Release Notes for the WebView2 SDK](../release-notes.md).

Reply:
adding hidden comment to the new table of 3 status rows, like:
try to only describe APIs as "stable" and "experimental", and only describe SDKs/packages as "release" and "prerelease"
I will try out such terminology usage in the edited spots, rather than creating a status called "Release".